proj-6/Final_Project_TEI_Complete.xml

```diff
@@ -0,0 +1,73 @@
+               <line>the <gap reason="illegible" />this <unclear cert="medium">charity</unclear>. Their <gap reason="illegible" /></line>
+               <line>a <gap reason="illegible" />which allowed them</line>
+               <line><gap reason="illegible" />themselves seems</line>
+               <line>their <unclear cert="high">least</unclear>disadvantage. Civi-</line>
+               <line>lization seems to make bright only</line>
+               <line>the superficial film of the eye. Most</line>
+               <line>men are wrecked upon their conscious-</line>
+	       <line>ness.</line>
+               <line>I had 3 chairs in my house -</line>               
+	       <line>1 for <unclear cert="low">solitude</unclear> - 2 for friends and</line>
+	       <line>3 - for society. When visitors came</line>
+	       <line>in larger and unexpected numbers</line>
+	       <line>there was but the <unclear cert="medium">3d</unclear> for them</line>
+	       <line>all - but they generally economised</line>
+	       <line>the room by standing up. <gap reason="illegible" /> -</line>
+	       <line>something how many great men</line>
+	       <line>and women a small house will</line>
+	       <line>contain. I had had 25 or 30</line>
+	       <line>souls and <add place="above line, pencil, prior word in parentheses">with their</add> bodies <unclear cert="medium">at one</unclear> under</line>
+	       <line>my roof - and yet so often parted</line>
+	       <line>without being aware that we had</line>
+	       <line>come very near to one another. <add place="above line, pencil, carrot"><unclear cert="low">Many</unclear> of</add> <unclear cert="low">the houses</unclear></line>
+	       <line><del rend="strikethrough in pencil">generally</del> with their <handShift medium="pencil"><add place="above line, carrot"><unclear cert="low">immense</unclear> apartments, their</add></handShift> huge halls and</line>
+	       <line><del rend="strikethrough in pencil">garrets</del> and cellars, <handShift medium="pencil"><add place="above line, carrot">for the image of wines and <gap reason="illegible" />is<gap reason="illegible" /> of hence</add></handShift> seem <unclear cert="low">to me ex-</unclear></line>
+	       <line>travagantly large for their in-</line>
+               <line><unclear cert="medium">habitants</unclear> <handShift medium="pencil"><add place="above line, carrot">they are <gap reason="illegible" />and<gap reason="illegible" />that the<gap reason="illegible" />are<gap reason="illegible" />them<gap reason="illegible" /><unclear cert="low">frequently</unclear><gap reason="illegible" />vermin that<gap reason="illegible" />therein</add>(</handShift>.<del rend="strikeout in pencil"><note><!--Left bracket and strikethrough are present in rust-coloured colored pencil or grease pencil, most likely from Franklin Sanborn after Thoreau's death--></note><unclear cert="low">One would</unclear> certainly</del></line>
```

Comment:
@soitgoes4 See our conversation in the Slack help channel from 27 April beginning 4:51 pm. There's still some work to do on this file's use of `handShift`. Don't forget you have a friend in `@rend`. :-)